configs/kubernetes

```diff
@@ -1,4 +1,4 @@
-DOCKER_REPO = sttts/k8s-publishing-bot
+DOCKER_REPO = gcr.io/k8s-staging-publishing-bot/k8s-publishing-bot
```

Comment:
can I push there?

Reply:
Yes, anyone here can push: https://github.com/kubernetes/k8s.io/blob/52d24be5d4211def73907cf600e43c80d53ea29c/groups/groups.yaml#L132-L136

Today, this is you, @dims and me.